src/android/com/yellow/ai/ymchat/YmChatCordova.java

```diff
@@ -98,6 +98,27 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
       case "sendEventToBot":
         sendEventToBot(args,callbackContext);
         return true;
+      case "setThemeBotName":
+        setThemeBotName(args,callbackContext);
+        return true;
+      case "setThemeBotDescription":
+        setThemeBotDescription(args,callbackContext);
+        return true;
+      case "setThemePrimaryColor":
+        setThemePrimaryColor(args,callbackContext);
+        return true;
+      case "setThemeSecondaryColor":
+        setThemeSecondaryColor(args,callbackContext);
+        return true;
+      case "setThemeBotBubbleBackgroundColor":
+        setThemeBotBubbleBackgroundColor(args,callbackContext);
+        return true;
+      case "setThemeBotIcon":
+        setThemeBotIcon(args,callbackContext);
+        return true;
+      case "setThemeBotClickIcon":
+        setThemeBotClickIcon(args,callbackContext);
+        return true;
```

Comment:
Do we not support another methods of Theme till now?

Reply:
Methods were added there, but handling of those methods were not there previously, so added the missing implementations